web-tester/src/main/java/com/axonivy/ivy/webtest/engine/WebAppFixture.java

```diff
@@ -167,8 +167,8 @@ public void resetVar(String name) {
    */
   public void config(String name, String value) {
     try {
-      var url = configRestUrl(CONFIG_PATH + "/" + name) + "?value=" + value;
-      sendRequest(HttpRequest.newBuilder(new URI(url)).POST(BodyPublishers.noBody()));
+      var url = configRestUrl(CONFIG_PATH + "/" + name);
```

Comment:
Is there no need to pass `name` in a url builder so that it is properly encoded?

Reply:
Yes would be better I think...
But less important I think, as the name normally has not special chars (yes I know there are some)

Reply:
But send the value in the body is anyway better I think, as the config technically also support to write multiline stuff etc...

Reply:
The problem is, currently I have no access to the `javax.ws.rs.core.UriBuilder` class in the webtester. Meaning I would need to add this dependecy to the web-tester